src/views/HomeView.vue

```diff
@@ -9,7 +9,19 @@
       </div>
 
       <div class="textarea-block">
-        <TextArea :value="outputTextarea" />
+        <TextArea :value="outputTextarea" :readonly="true" />
+        <Button
+          v-clipboard:copy="outputTextarea"
+          v-clipboard:success="copySuccess"
+          v-clipboard:error="copyError"
+          :class="copyBtnClass"
+          @click.once="isCopyMessage = !isCopyMessage"
```

Comment:
Do you really need onclick here?

Reply:
Yep, if it's about button. Because I'll reuse component and can be different functions to call onclick